File: ruby/vristo-ruby-starter/app/controllers/pages_controller.rb
class PagesController < ApplicationController

    def pages_knowledge_base
        render layout: "default", :template =>'pages/knowledge-base'
    end

    def pages_contact_us
        render layout: "auth", :template =>'pages/contact-us'
    end

    def pages_faq
        render layout: "default", :template =>'pages/faq'
    end

    def pages_coming_soon
        render layout: "auth", :template =>'pages/coming-soon'
    end

    def pages_error404
        render layout: "auth", :template =>'pages/error404'
    end

    def pages_error500
        render layout: "auth", :template =>'pages/error500'
    end

    def pages_error503
        render layout: "auth", :template =>'pages/error503'
    end

    def pages_maintenence
        render layout: "auth", :template =>'pages/maintenence'
    end
end

File: cakephp/vristo-cakephp-starter/src/Controller/PagesController.php
<?php
declare(strict_types=1);

namespace App\Controller;

class PagesController extends AppController
{
    public function pages_knowledge_base()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/pages/knowledge-base');
    }

    public function pages_contact_us()
    {
        $this->viewBuilder()->setLayout('auth');
        return $this->render('/pages/contact-us');
    }

    public function pages_faq()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/pages/faq');
    }

    public function pages_coming_soon()
    {
        $this->viewBuilder()->setLayout('auth');
        return $this->render('/pages/coming-soon');
    }

    public function pages_error404()
    {
        $this->viewBuilder()->setLayout('auth');
        return $this->render('/pages/error404');
    }

    public function pages_error500()
    {
        $this->viewBuilder()->setLayout('auth');
        return $this->render('/pages/error500');
    }

    public function pages_error503()
    {
        $this->viewBuilder()->setLayout('auth');
        return $this->render('/pages/error503');
    }

    public function pages_maintenence()
    {
        $this->viewBuilder()->setLayout('auth');
        return $this->render('/pages/maintenence');
    }

}
